### DISCUSSION TITLE
Whitelisting folders

### DISCUSSION THREAD

**Discussion by thoresson** [brianpetro/obsidian-smart-connections#309] [Ideas]
Hi!

At the moment, it's possible to blacklist folders from being included in Smart Connection's analysis of a vault. Has whitelist approach being considered? I know for sure which two folders I want to include, and all others (current and future) I want to exclude. Somewhat afraid that I will forget to add new folders to the exclusion list down the road.

/Anders

**Comment by pictoys** [comment]
+1 from me, I'd find this an essential feature. there is too much in my vault i would not want to share.
